test/upgrade/continual.go

```diff
@@ -38,6 +40,17 @@ func BrokerContinualTests() []pkgupgrade.BackgroundOperation {
 	}
 }
 
+// ChannelContinualTests returns background operations to test KafkaChannel
+// functionality in continual manner during the whole upgrade and downgrade
+// process asserting that all events are propagated.
+func ChannelContinualTests() []pkgupgrade.BackgroundOperation {
+	// don't want to run the Broker-Backed-By-KafkaChannel tests,
+	// so, we don't call the test method directly
```

Comment:
why not ? my naive thinking says that it increases coverage.

Reply:
I think proper way to use a broker, if you have this repository is to use KafkaBroker and not MTBroker

Reply:
+1 but what if there are existing users using KafkaChannel based brokers, is this setup still supported?

Reply:
I don't know...

I would prefer @matzew to have a look at that

Reply:
I will enable it know and see if it passes w/o additional work. If it does I will unhold.
If there's work, we should discuss if we want to support this.